apps/deploy-web/src/components/new-deployment/NewDeploymentContainer.tsx

```diff
@@ -80,7 +77,9 @@ export const NewDeploymentContainer: FC = () => {
     if ("config" in template && (template.config?.ssh || (!template.config?.ssh && hasComponent("ssh")))) {
       toggleCmp("ssh");
     }
-    const isRemoteYamlImage = isImageInYaml(template?.content as string, getTemplateById(CI_CD_TEMPLATE_ID)?.deploy);
+
+    const cicdTemplate = getTemplateById(CI_CD_TEMPLATE_ID);
+    const isRemoteYamlImage = isImageInYaml(template?.content as string, cicdTemplate?.deploy);
```

Comment:
It seems like this should yield the same result as previously?

Reply:
If the templates are not loaded it returns so it shouldn't happen. Anyways I don't see an issue releasing that.